dashboard/app/controllers/resources_controller.rb

```diff
@@ -3,7 +3,7 @@ class ResourcesController < ApplicationController
 
   # GET /resourcesearch/:q/:limit
```

Comment:
nit: remove `q` and `limit` from this comment, to match the new definition of this route?